test/corpus/expr/closure.nu

```diff
@@ -163,49 +172,22 @@ closure-007-record-value
   (pipeline
     (pipe_element
       (val_record
-        (record_entry
-          (identifier)
-          (val_closure
-            (parameter_pipes
-              (parameter
-                (identifier)))
-            (pipeline
-              (pipe_element
-                (expr_binary
-                  (val_variable
-                    (identifier))
-                  (val_number))))))
-        (record_entry
-          (identifier)
-          (val_closure
-            (pipeline
-              (pipe_element
-                (val_string)))))))))
-
-=====
-closure-008-while
```

Comment:
was closure-008-while just left out because it's a dupe or was this accidental?

Reply:
Accidental thanks - I will go through the diff once again to make sure I haven't forgotten anything. But none of the test should be removed, they should only get changes.

Reply:
I think everything should be fine now - I went through the test diffs & I don't see anymore lost tests.